packages/client/webpack.config.js

```diff
@@ -27,5 +27,9 @@ module.exports = () => {
     },
     mode: 'production',
     plugins: WITH_REPORT ? [new BundleAnalyzerPlugin()] : [],
+    performance: {
+      maxEntrypointSize: 394_000,
+      maxAssetSize: 394_000,
```

Comment:
the seem not working?
![image](https://user-images.githubusercontent.com/10594507/141252575-47a58232-28eb-47f6-99be-e0f9d06f337c.png)

Reply:
playground package exceed its limit, not client

Reply:
My bad.